build-system/tasks/coverage-map.js

```diff
@@ -0,0 +1,98 @@
+/**
+ * Copyright 2020 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+const argv = require('minimist')(process.argv.slice(2));
+const fs = require('fs');
+const log = require('fancy-log');
+const puppeteer = require('puppeteer');
+const {exec} = require('../common/exec');
+
+const coverageJsonName = argv.json || 'out.json';
+const testUrl =
+  argv.url || 'http://localhost:8000/examples/everything.amp.html';
+
+async function compile() {
+  await exec(`gulp dist`);
+  // The question: How to cd to 'amphtml/dist' from anywhere in 'amphtml'?
```

Comment:
In addition to this, when we run `gulp serve`, a another terminal will be needed to run puppeteer.

Reply:
I believe you can use `startServer` function from serve task to run in the background

Reply:
I see. I'd assume in the same manner that I could just run exported `dist` instead of calling `exec`.
What do you mean by running in the background? Does `serve` not take up a terminal? I would always launch another terminal to run in parallel to `serve`.

Reply:
@kevinkimball is correct. `gulp serve` is the standalone task to launch a server and needs its own terminal. However, you can also call `startServer()` inside another task and launch the server in-process. In this case, it doesn't need its own terminal. The default `gulp` task does this, as do tasks like `gulp unit|integration|e2e|visual-diff`.

Reply:
I ended using `startServer`, passing `compiled: true` inside `options`. As it turns out, `source-map-explorer` does not like unminified JS.